builds/android/metadata/pt-BR/title.txt

```diff
@@ -1 +1 @@
-EasyRPG para RPG Maker 2000
\ No newline at end of file
+Reprodutor EasyRPG
```

Comment:
Not sure about this rename, but since I do not speak any language added here...

![reaction](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExcDZiMWUwcHBjMjYwYnBrNmhxb2xnY2pvaHdiMmNxc3N3YXFrMTBzaCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/2d98nRiVVB6HS/giphy.gif)